bundles/net.tourbook/src/net/tourbook/ui/views/rawData/DialogReimportTours.java

```diff
@@ -651,9 +932,7 @@ public String getSubTaskText(final TourData savedTourData) {
 
       } else {
 
-         // re-import SELECTED tours
-
-         RawDataManager.getInstance().actionReimportSelectedTours(tourValueTypes, _tourViewer, skipToursWithFileNotFound);
+         actionReimportSelectedTours(tourValueTypes, _tourViewer, skipToursWithFileNotFound);
```

Comment:
Comment is lost

Reply:
I removed it because i thought that the name of the function `actionReimportSelectedTours(`) is obvious enough.

As Robert M [says very well](https://medium.com/@huytrongnguyen1985/lessons-learnt-from-the-clean-code-robert-c-martin-cecbe2b09139) : `Comments should say things that the code cannot say for itself.`